docs/conf.py

```diff
@@ -45,9 +46,13 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
-
+html_theme = 'sphinx_rtd_theme'
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
\ No newline at end of file
+html_static_path = ['_static']
+
+# Add Fiware css stylesheet
+def setup(app):
+    app.add_stylesheet('css/fiware_readthedocs.css')
```

Comment:
Can't we reference an external stylesheet (and thus avoid to maintain a copy on our own)?

Reply:
done in 3f9668593213d7b2648800b8deb2b32229567d21